### PR TITLE
Change About section

### LINKED ISSUE #86: About Changer (Idea)
1. Change the About every second to looks like is dancing
2. Change the About every 30 min (you choose) with random phrases from Internet.

### PR DESCRIPTION
**Pull request by Rits1272**
Fixes #86
This script changes the about section of your whatsapp account. You can change the about once in every second or once in 30 min. 
**The script fetches an api that gives the total coronavirus cases around the globe and changes your about section with that.**
 
How to use:
> python3 -m wplay -wabt OR python3 -m wplay --wabout

You then need to give a choice to either change your about in 1 sec or 30 min

### CODE REVIEW

**Comment by Rits1272** [comment]
Please review @rpotter12 @xandao6

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/changeAbout.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221653&bid=17120782)

File: wplay/utils/helpers.py [hunk 1]
```diff
@@ -33,6 +33,9 @@
     'chats_groups_messages_elements': '#side > div:last-child > div > div > div > div',
     'contact_element': 'span > span > span[class^="matched-text"]',
     'group_element': 'div:last-child > div:first-child > div:first-child > div > span > span[class^="matched-text"]',
+    'profile_photo_element': '#side > header > div > div > img',
+    'about_edit_button_element': '#app > div > div > div > div > span > div > div > div > div:nth-child(4) > div > div > span > div > span',
+    'about_text_area': '#app > div > div > div > div > span > div > div > div > div:nth-child(4) > div > div > div > div'
```
Comment (xandao6) [inline wplay/utils/helpers.py]: I suggest you to remove the specific array access `nth-child(4)` because is very easy to broke.

Comment (xandao6) [reply wplay/utils/helpers.py]: If it's possible.

File: wplay/changeAbout.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from wplay.utils.helpers import whatsapp_selectors_dict
+from wplay.utils import browser_config
+import time
+import sys
+import requests
+
+async def changeAbout():
+    page, _ = await browser_config.configure_browser_and_load_whatsapp()
+    choice = input("Choose 1 for changing About section every second and 2 for changing About in every 30 min : ")
```
Comment (xandao6) [inline wplay/changeAbout.py]: We can let him choose the time!

Comment (xandao6) [reply wplay/changeAbout.py]: But it's okay for now, we can upgrade later.

File: wplay/changeAbout.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        for _ in range(140):
+            await page.keyboard.press('Backspace')
+        await page.type(whatsapp_selectors_dict['about_text_area'], get_live_cases())
+        await page.keyboard.press('Enter')
+        if choice == '1':
+            time.sleep(1)
+        else:
+            time.sleep(1800)
+
+def get_live_cases():
```
Comment (xandao6) [inline wplay/changeAbout.py]: We can envolve a lot this area.

Comment (xandao6) [reply wplay/changeAbout.py]: Currently only bring us coronavirus news, but the idea is to let the used define what we want.

**Comment by xandao6** [comment]
@rpotter12 take a look, this PR is quite interesting.

File: wplay/changeAbout.py (hunk 3 above)
Comment (Rits1272) [reply wplay/changeAbout.py]: Just to make it a little interesting. If you allow we can change it for later

File: wplay/utils/helpers.py (hunk 1 above)
Comment (Rits1272) [reply wplay/utils/helpers.py]: I will try @xandao6 :)

**Pull request merged**